palisade-service/src/main/resources/application.yaml

```diff
@@ -48,3 +48,11 @@ feign:
         readTimeout: 5000
         loggerLevel: basic
     default-config: default
+
+# Default development & testing static config, not suitable for production
+web:
+  client:
+    audit-service: http://localhost:8081
```

Comment:
potential port change to avoid port conflicts

Reply:
done